lib/node_modules/@stdlib/_tools/scaffold/math-special-unary/scripts/dtype_ndarray_mappings.json

```diff
@@ -0,0 +1,12 @@
+{
+	"input_dtypes": {
```

Comment:
As discussed during a previous call, you can build out this table for completeness to include other dtype mappings.